docs/installation/config/env_configuration.rst

```diff
@@ -89,7 +89,7 @@ Optional
 * ``RELEASE``: The version number or commit hash of the application (this is also sent to Sentry).
```

Comment:
Can you also make sure `bin/generate_envvar_docs.sh` uses the new path `docs/installation/config/env_configuration.rst`?